bot/exts/evergreen/emoji_count.py

```diff
@@ -17,73 +19,71 @@ class EmojiCount(commands.Cog):
     def __init__(self, bot: commands.Bot):
         self.bot = bot
 
-    def embed_builder(self, emoji: dict) -> discord.Embed:
+    @staticmethod
+    def embed_builder(emoji: dict) -> Tuple[discord.Embed, List[str]]:
         """Generates an embed with the emoji names and count."""
         embed = discord.Embed(
             color=Colours.orange,
             title="Emoji Count",
             timestamp=datetime.datetime.utcnow()
         )
+        msg = []
 
         if len(emoji) == 1:
-            for key, value in emoji.items():
-                embed.description = f"There are **{len(value)}** emojis in the **{key}** category"
-                embed.set_thumbnail(url=random.choice(value).url)
+            for category_name, category_emojis in emoji.items():
+                msg.append(f"There is **{len(category_emojis)}** emoji in the **{category_name}** category")
```

Comment:
this string need to dynamic based on singular or plural nature of `len(category_emojis)`
```suggestion
                msg.append(
                    f"There {'is' if len(category_emojis)==1 else 'are'} **{len(category_emojis)}** "
                    f"{'emoji' if len(category_emojis)==1 else 'emojis'} in the **{category_name}** category"
                )
```